prov/sockets/src/sock_ep_msg.c

```diff
@@ -1093,11 +1091,6 @@ static void *sock_pep_listener_thread(void *data)
 						conn_req->hdr.msg_id);
 
 		switch (conn_req->hdr.type) {
-		case SOCK_CONN_ACK:
-			SOCK_LOG_DBG("Received SOCK_CONN_ACK\n");
-			sock_ep_cm_handle_ack(&pep->cm, &conn_req->hdr);
-			break;
-
```

Comment:
Thanks! I have changed the signature of <code>sock_ep_cm_handle_ack</code> to take <code>struct sock_ep</code> instead of <code>sock_ep->cm</code> and this case wouldn't work with the same signature as it's a passive ep case. That's why I removed the unused case in the same patch. Sorry for the confusion.